index.jsx

```diff
@@ -1,9 +1,13 @@
 require("./node_modules/materialize-css/dist/css/materialize.min.css")
 require("./node_modules/materialize-css/dist/js/materialize.js");
 import React from 'react';
+import { render } from 'react-dom'
```

Comment:
Let's stick with the namespacing of `ReactDOM.render` for now.  Why the project-wide change in the first place?